.eleventy.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+module.exports = function(eleventyConfig) {
+  eleventyConfig.addPassthroughCopy('docs/css');
+  eleventyConfig.addPassthroughCopy('docs/js');
+  eleventyConfig.addPassthroughCopy('docs/images');
+  eleventyConfig.addPassthroughCopy('docs/CNAME');
+  eleventyConfig.addPassthroughCopy('docs/_headers');
+  eleventyConfig.addPassthroughCopy('docs/favicon.ico');
+
+  eleventyConfig.addPassthroughCopy('docs/example');
```

Comment:
Why is "doc/example" not named "doc/examples"?

Reply:
Good question. My best guess is because there's only one

Reply:
No, there are multiple examples...

Reply:
lol